File: pgm/ABC_Visual_Basic/ABC_159_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim N, M As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        N = arr(0) : M = arr(1)
        Console.WriteLine( _
            Math.Ceiling(N * (N - 1) / 2) + Math.Ceiling(M * (M - 1) / 2))
    End Sub
End Class

File: pgm/ABC_C#/ABC_159_A.cs
using System;
// using System.Collections.Generic;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int N = int.Parse(arr[0]), M = int.Parse(arr[1]);
        Console.WriteLine(N * (N - 1) / 2 + M * (M - 1) / 2);
    }
}